<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowAdmin</class>
 <widget class="QMainWindow" name="MainWindowAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>811</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="reservation_tab">
     <attribute name="title">
      <string>Reservation</string>
     </attribute>
     <widget class="QLineEdit" name="table_priceEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Table price</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="begin_timeEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="time">
       <time>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
     <widget class="QTableWidget" name="reservation_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>751</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Order ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Duration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Table type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Table ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deposit</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>5</month>
        <day>27</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="table_idEdit">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Table ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QComboBox" name="table_typeBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Table (two)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Table (four)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Box</string>
       </property>
      </item>
     </widget>
     <widget class="QTimeEdit" name="end_timeEdit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="time">
       <time>
        <hour>13</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
     <widget class="QLabel" name="add_promptLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Click Add to add a reservation</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Food reservation</string>
     </attribute>
     <widget class="QTableWidget" name="pair_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>751</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Pair ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reservation ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Food ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Food name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="pair_promptLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Click Delete to delete a food reservation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pair_deleteButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Food</string>
     </attribute>
     <widget class="QTableWidget" name="food_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>751</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Food ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="add_food_promptLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Click Add to add food</string>
      </property>
     </widget>
     <widget class="QPushButton" name="food_addButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="food_deleteButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="foodnameEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Food name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="foodpriceEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fooddescriptionEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Description</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user_tab">
     <attribute name="title">
      <string>User</string>
     </attribute>
     <widget class="QTableWidget" name="user_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>751</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nickname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Phone Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Password</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="user_deleteButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>751</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Other</string>
     </attribute>
     <widget class="QPushButton" name="user_id_resetButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset user ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reservation_id_resetButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>68</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset reservation ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="administrator_id_resetButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset administrator ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reservation_table_clearButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear reservation Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="user_table_clearButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear user Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="administrator_table_clearButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear administrator Table</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>These buttons are typically used for testing.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reservation_table_createButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create reservation Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="administrator_table_createButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>190</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create administrator Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="user_table_createButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create user Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pair_id_resetButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset pair ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pair_table_clearButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear pair Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pair_table_createButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>310</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create pair Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="food_id_resetButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset food ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="food_table_clearButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear food Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="food_table_createButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create food Table</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Themes</string>
      </property>
     </widget>
     <widget class="QComboBox" name="theme_selectBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Default</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BlackNBlue</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="up_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>600</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
